defaults/main.yml

```diff
@@ -34,3 +34,11 @@ haproxy_enable_syslog: false
 haproxy_use_graylog_json_logging: false
 haproxy_graylog_server:
 haproxy_graylog_server_port:
+
+# Prometheus exporter support for haproxy v1.8
+haproxy_external_exporter_enabled: false
+haproxy_exporter_download_url: https://github.com/prometheus/haproxy_exporter/releases/download/v0.12.0/haproxy_exporter-0.12.0.linux-amd64.tar.gz
```

Comment:
Would it be better to get the version from its own variable?

Reply:
I didn't care to make this too flexible. This isn't necessary in haproxy v2+ so I don't see this being around forever.

Reply:
fair enough